Author: Simon Frankenberger <simon-alpine@fraho.eu>
Upstream-Issue: https://github.com/AsamK/signal-cli/issues/1022
Subject: Change the UserAgent in API calls to a newer version

Since 2022-10-05, about 20:00 UTC the signal API backend checks the calling client version.
As the signal-cli used a pretty old version, this triggered some DeprecatedVersionException
which broke the application.

--- old/src/main/java/org/asamk/signal/BaseConfig.java
+++ new/src/main/java/org/asamk/signal/BaseConfig.java
@@ -5,7 +5,7 @@
     public final static String PROJECT_NAME = BaseConfig.class.getPackage().getImplementationTitle();
     public final static String PROJECT_VERSION = BaseConfig.class.getPackage().getImplementationVersion();
 
-    final static String USER_AGENT_SIGNAL_ANDROID = "Signal-Android/5.22.3";
+    final static String USER_AGENT_SIGNAL_ANDROID = "Signal-Android/5.51.7";
     final static String USER_AGENT_SIGNAL_CLI = PROJECT_NAME == null
             ? "signal-cli"
             : PROJECT_NAME + "/" + PROJECT_VERSION;
